tools/integration_tests/write_large_files/write_large_files_test.go

```diff
@@ -31,8 +31,8 @@ func TestMain(m *testing.M) {
 
 	setup.ExitWithFailureIfBothTestBucketAndMountedDirectoryFlagsAreNotSet()
```

Comment:
We can simplify the logic here - bucket is mandatory now, checking that will be enough.

Reply:
For operation package we can still pass only mountedDirectory flag.